projects/components/src/paginator/paginator.component.ts

```diff
@@ -232,7 +228,7 @@ export class PaginatorComponent implements OnChanges, PaginationProvider {
       return this.pageSize;
     }
 
-    if (this.totalItems === PaginatorTotalCode.Unknown || this.totalItems === PaginatorTotalCode.Last) {
+    if (this.totalItems === PaginatorTotalCode.Unknown) {
```

Comment:
Sorry I missed this before approving PaginatorTotalCode.Last should be removed entirely.

Reply:
PaginatorTotalCode.Last has been completely removed from the file